conformance/reports/v1.0.0/cilium.yaml

```diff
@@ -24,6 +24,19 @@ profiles:
         Passed: 12
         Skipped: 0
       summary: ""
+      supportedFeatures:
```

Comment:
Thanks for adding this, I just update gateway-api version in cilium/cilium to pick up the new format.

Here is the extra block in new report generated by CI https://github.com/cilium/cilium/actions/runs/7718271763

```
    supportedFeatures:
    - HTTPRouteBackendTimeout
    - HTTPRouteHostRewrite
    - HTTPRouteMethodMatching
    - HTTPRouteParentRefPort
    - HTTPRoutePathRedirect
    - HTTPRoutePathRewrite
    - HTTPRoutePortRedirect
    - HTTPRouteQueryParamMatching
    - HTTPRouteRequestMirror
    - HTTPRouteRequestMultipleMirrors
    - HTTPRouteRequestTimeout
    - HTTPRouteResponseHeaderModification
    - HTTPRouteSchemeRedirect
```

Reply:
Apart from sorting order, we can add HTTPRouteParentRefPort into this list.

Reply:
gotcha, updated it now. Thank you for the update!